### PR TITLE
[ext-doc] Favicon API follow up tasks

### LINKED ISSUE #4853: [ext-docs] Favicon API follow-up tasks
When PR#4852 is merged, the following pages need to link to the new doc:

- [ ] Add new doc link to favicon permission description in the [permission warning table](https://developer.chrome.com/docs/extensions/mv3/permission_warnings/#permissions_with_warnings).
- [ ] In [Manifest Overview](https://developer.chrome.com/docs/extensions/mv3/intro/mv3-overview/), link [favicon API list item](https://developer.chrome.com/docs/extensions/mv3/intro/mv3-overview/#:~:text=New%20favicon%20API%3A%20this%20new%20JavaScript%20API%20replaces%20%22chrome%3A//favicons%22%20and%20gives%20developers%20a%20way%20to%20retrieve%20websites%27%20favicons.) to new doc. 
- [ ] Add favicon migration instructions to [Migrate to MV3](https://developer.chrome.com/docs/extensions/mv3/mv3-migration/).

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #4853

### CODE REVIEW

File: site/en/docs/extensions/mv3/favicon/index.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
 layout: "layouts/doc-post.njk"
 title: "Fetching favicons"
-seoTitle: "Fetching with favicons in Chrome extensions"
+seoTitle: "Fetching favicons in Chrome extensions"
 date: 2023-01-11
 # updated: 2023-02-14
-description: How to get a website's favicon in a Chrome extension.
+description: How to get a website's favicon.
```
Comment (jpmedley) [inline site/en/docs/extensions/mv3/favicon/index.md:7]: I like this change. Information that's implicit by context has limited utility, I think.

File: site/en/docs/extensions/mv3/favicon/index.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ chrome-extension://EXTENSION_ID/_favicon/?pageUrl=EXAMPLE_URL&size=FAV_SIZE
 : The URL of the favicon's website.
 
 `FAV_SIZE`
-: The size of the favicon. For example: 16, 32, or 64.
+: The size of the favicon. The most common size is 16 pixels.
```
Comment (jpmedley) [inline site/en/docs/extensions/mv3/favicon/index.md:30]: I agree with adding your line. Should we not list what the common sizes are? Is that information somewhere else?

File: site/en/docs/extensions/mv3/intro/mv3-overview/index.md [hunk 3]
```diff
@@ -145,16 +128,16 @@ There are a number of other changes introduced in Manifest V3:
...
-* **Dynamic content scripts:** the new [Scripting API][1] lets extensions register and unregister content scripts at runtime.
-* **New favicon API:** this new JavaScript API replaces "chrome://favicons" and gives  developers a way to retrieve websites' favicons.
-
-Look for announcements of these and other Manifest V3 features as they become available.
-
-[1]: /docs/extensions/reference/scripting/
-[2]: /docs/extensions/reference/storage/#property-session
-[3]: https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise
-[4]: /docs/extensions/mv3/promises#compare-to-callback
+* [A new in-memory StorageArea][storage-session] that can be used to store values in-memory across service worker restarts.
```
Comment (jpmedley) [inline site/en/docs/extensions/mv3/intro/mv3-overview/index.md:131]: Please make items in the list match each other grammatically.
"Express coordinate ideas in similar form." (Strunk and White, rule II-19.)

File: site/en/docs/extensions/mv3/intro/mv3-overview/index.md [hunk 4]
```diff
@@ -145,16 +128,16 @@ There are a number of other changes introduced in Manifest V3:
...
+* [Dynamic content scripts][cs-runtime]: the new [Scripting API][doc-scripting] lets extensions register and unregister content scripts at runtime.
```
Comment (jpmedley) [inline site/en/docs/extensions/mv3/intro/mv3-overview/index.md:132]: Per the Chicago Manual of Style (Google standard), starting lower case after the colon is correct. The items you didn't touch don't match. The easier thing is probably to make your new items upper case after the colon. Your choice.

**Review by jpmedley — CHANGES_REQUESTED** [review]
Looking good!

File: site/en/docs/extensions/mv3/intro/mv3-overview/index.md (hunk 3 above)
Comment (AmySteam) [reply site/en/docs/extensions/mv3/intro/mv3-overview/index.md:131]: 👍

File: site/en/docs/extensions/mv3/intro/mv3-overview/index.md (hunk 4 above)
Comment (AmySteam) [reply site/en/docs/extensions/mv3/intro/mv3-overview/index.md:132]: I'll make the new items upper case 👍

File: site/en/docs/extensions/mv3/favicon/index.md (hunk 2 above)
Comment (AmySteam) [reply site/en/docs/extensions/mv3/favicon/index.md:30]: I'm going by [MDN's favicon doc](https://developer.mozilla.org/en-US/docs/Glossary/Favicon) which mentions the most common size.

Comment (jpmedley) [reply site/en/docs/extensions/mv3/favicon/index.md]: I'm not objecting to adding that. I'm asking if you should say anything about other sizes.

Comment (AmySteam) [reply site/en/docs/extensions/mv3/favicon/index.md]: I defer to your judgment on this matter. Please make a suggestion.

Comment (jpmedley) [reply site/en/docs/extensions/mv3/favicon/index.md]: I misread this. This is 'favorite size' not 'icon sizes'. That's why I was asking for more.

**Review by jpmedley — APPROVED** [review]

**Pull request merged**